src/javascript/app/ng-wig/ngWigToolbarProvider.js

```diff
@@ -1,23 +1,39 @@
 angular.module('ngWig').provider('ngWigToolbar', function () {
 
   var buttonLibrary = {
-    list1: {title: 'Unordered List', command: 'insertunorderedlist', styleClass: 'nw-button--unordered-list'},
-    list2: {title: 'Ordered List', command: 'insertorderedlist', styleClass: 'nw-button--ordered-list'},
-    bold: {title: 'Bold', command: 'bold', styleClass: 'nw-button--bold'},
-    italic: {title: 'Italic', command: 'italic', styleClass: 'nw-button--italic'},
-    link: {title: 'Link', command: 'createlink', styleClass: 'nw-button--link'}
+    list1: {title: 'Unordered List', command: 'insertunorderedlist', styleClass: 'nw-button--unordered-list', isSimple: true},
+    list2: {title: 'Ordered List', command: 'insertorderedlist', styleClass: 'nw-button--ordered-list', isSimple: true},
+    bold: {title: 'Bold', command: 'bold', styleClass: 'nw-button--bold', isSimple: true},
+    italic: {title: 'Italic', command: 'italic', styleClass: 'nw-button--italic', isSimple: true},
+    link: {title: 'Link', command: 'createlink', styleClass: 'nw-button--link', isSimple: true}
```

Comment:
+1
it's better to put here param like "complex"(or better name) and it's false by default